content/en/methods/media.md

```diff
@@ -366,7 +366,7 @@ Attachment is not owned by you or does not exist
 
 ---
 
-## (DEPRECATED) Upload media as an attachment {#v1}
+## Upload media as an attachment (v1) {{%deprecated%}} {#v1}
```

Comment:
`(v1)` should probably be replaced with `(non-async)` or otherwise removed

Reply:
it is the `/api/v1/` endpoint? I don't really mind, but I based this on the others having (v2) in places.